### PR TITLE
Scripts: updated dev_start.sh

### PR DESCRIPTION
**Pull request by storypku**

### CODE REVIEW

File: docker/scripts/dev_start.sh [hunk 1]
```diff
@@ -278,10 +276,6 @@ function setup_devices_and_mount_local_volumes() {
     setup_device
 
     local volumes="-v $APOLLO_ROOT_DIR:/apollo"
-    local teleop="${APOLLO_ROOT_DIR}/../apollo-teleop"
```
Comment (storypku) [inline docker/scripts/dev_start.sh:281]: No longer mount local `teleop`  repo for the OSS version.

Comment (Capri2014) [reply docker/scripts/dev_start.sh:281]: why skip the teleop mounting step?

Comment (storypku) [reply docker/scripts/dev_start.sh:281]: > why skip the teleop mounting step?

It was mounted in our internal `dev_start.sh`, and teleop was not public visible.
Maybe did the mount here target teleop guys ?

Comment (Capri2014) [reply docker/scripts/dev_start.sh:281]: > > why skip the teleop mounting step?
> 
> It was mounted in our internal `dev_start.sh`, and teleop was not public visible.
> Maybe did the mount here target teleop guys ?

yes, to make things easier for Udelv folks.

Comment (storypku) [reply docker/scripts/dev_start.sh]: OK. Restored. Sorry for my unthoughtfulness.

File: docker/scripts/dev_start.sh [hunk 2]
```diff
@@ -239,14 +239,12 @@ function determine_dev_image() {
...
     # If no custom version specified
     if [ -z "${version}" ]; then
-        if [ "${USE_LOCAL_IMAGE}" = "yes" ]; then
-            version="local_dev"
-        elif [ "${TARGET_ARCH}" = "x86_64" ]; then
+        if [ "${TARGET_ARCH}" = "x86_64" ]; then
             version="${VERSION_X86_64}"
         elif [ "${TARGET_ARCH}" = "aarch64" ]; then
             version="${VERSION_AARCH64}"
         else
```
Comment (storypku) [inline docker/scripts/dev_start.sh:246]: In this PR: when `dev_start.sh -l` was run,  It no longer searches for the `local_dev` image. Instead, image tag for that arch is used.  If someone do need the `local_dev` tag, he/she should just run `dev_start.sh -l -t local_dev`.

File: scripts/mobileye.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+TOP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+source "${TOP_DIR}/scripts/apollo_base.sh"
+
+# run_customized_path command_name module_name
+run_customized_path drivers/mobileye mobileye "$@"
```
Comment (storypku) [inline scripts/mobileye.sh:23]: mobileye restored.

**Comment by Capri2014** [comment]
@storypku Looks like in latest commit you did not submit all changes?

**Comment by storypku** [comment]
> @storypku Looks like in latest commit you did not submit all changes?

Restored.  pushed on the wrong machine. Maybe I need some rest now.

**Pull request merged**